PennMobile.xcodeproj/project.pbxproj

```diff
@@ -2125,7 +2125,7 @@
 				CODE_SIGN_ENTITLEMENTS = PennMobile/PennMobile.entitlements;
 				CODE_SIGN_IDENTITY = "iPhone Developer";
 				CODE_SIGN_STYLE = Automatic;
-				CURRENT_PROJECT_VERSION = 6511;
+				CURRENT_PROJECT_VERSION = 6520;
```

Comment:
I'm not too familiar with the project.pbxproj file but do you know what caused this change as well as the ones below?

Reply:
I actually meant to update this in a separate branch, but got lazy / didn't want to forget. Either way, it's probably better to make this version change at the very end.